### ISSUE TITLE
Loss graphics

### ISSUE THREAD

**Issue by ashkanaev** [hfslyc/AdvSemiSeg#21]
Hello!
During training loss_D value decrease to ~0.2 and segmentation network cant learn nothing. Does it mean that discriminator dominate under generator? what range of loss_d value is normal(for classical gan is 0.69) )? Maybe it's possible to attach some loss graphics?

**Comment by hfslyc** [comment]
HI,

I would say it's usually on the range of 0.2-0.4. I currently don't have the loss graph. But you can train our network on pascal images for the loss comparison.
